public/app/features/integrated-alerting/components/NotificationChannel/AddNotificationChannelModal/AddNotificationChannelModal.test.tsx

```diff
@@ -1,14 +1,55 @@
 import React from 'react';
-import { mount } from 'enzyme';
+import { mount, ReactWrapper } from 'enzyme';
 import { act } from 'react-dom/test-utils';
 import { dataQa } from '@percona/platform-core';
 import { AddNotificationChannelModal } from './AddNotificationChannelModal';
 import { TYPE_OPTIONS } from './AddNotificationChannel.constants';
 import { notificationChannelStubs } from '../__mocks__/notificationChannelStubs';
+import { NotificationChannelType, PagerDutylNotificationChannel } from '../NotificationChannel.types';
 
 jest.mock('../NotificationChannel.service');
 jest.mock('app/core/app_events');
 
+/**
+ * Return this form's "Add" button
+ */
```

Comment:
I agree, no need to comment things that derive from the code. In general it's much better to spend time on writing readable code rather than commenting what's obvious.

Reply:
Agreed 👍